libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -52,6 +52,19 @@
 // motor update rate
 #define AP_MOTORS_SPEED_DEFAULT     490 // default output rate to the motors
 
+// a class that AP_MotorMask can be typedef'd to to ensure type-safety:
```

Comment:
I'm sure it's on purpose but it's a bit odd to have commented out code like this

Reply:
The comment there explains what it's for.  I thought it better to leave it available but commented out so it was easily available.

I could leave it commented in - or remove it.